mongoose/RestaurantSchema.ts

```diff
@@ -20,7 +20,7 @@ import Restaurant from "../models/Restaurant";
  */
 const RestaurantSchema = new mongoose.Schema<Restaurant>({
     name: {type: String, required: true},
-    ownedBy: {type: mongoose.Schema.Types.ObjectId, ref: "UserModel"},
+    ownedBy: {type: mongoose.Schema.Types.ObjectId, ref: "UserModel", required: true},
     bannerPicture: String,
```

Comment:
As mentioned in React project - I think it would be nice to add default restaurant banner and profile pic here

Reply:
Set default value for images.